index.js

```diff
@@ -256,6 +256,10 @@
     };
   };
 
+  var compose3 = function(f, g, x) {
+    return f(g(x));
+  };
+
```

Comment:
It goes to show how important we find composition, that we have four nearly identical implementations of it in a single code-base. ;)